DataCapturing/Source/Model/Altitude.swift

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 Cyface GmbH
+ *
+ * This file is part of the Cyface SDK for iOS.
+ *
+ * The Cyface SDK for iOS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface SDK for iOS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface SDK for iOS. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+
+/**
+ A struct to wrap all the information associated with a measured altitude provided by an altimeter.
+
+ - Author: Klemens Muthmann
+ */
+struct Altitude {
+    /// The relative altitude change since the last measured value.
+    let relativeAltitude: Double
+    /// The currently measured pressure.
+    let pressure: Double
+    /// The time this was measured.
+    let timestamp: Date
```

Comment:
Maybe add unit information here.

Reply:
I don't know yet. I will add that as soon as the SDK is connected here. This is currently only here to show some numbers on the UI. Doesn't matter which.